recipes/genform/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "r8" %}
+{% set name = "genform" %}
+{% set sha256 = "d9199aa9c13c6aa68025b15e4d30a4e445a265ce828b3314103307dc7b6c13b5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+#  skip: True  # [osx]
+
+source:
+  # download is a stable link for https://sourceforge.net/code-snapshots/svn/g/ge/genform/code/genform-code-r8-trunk.zip
+  url: https://depot.galaxyproject.org/software/genform/genform_r8_src_all.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+# genform without parameter or w help has exit code 1, so we need test data
```

Comment:
If you want to test whether the help text is printed correctly, you could do something like
```yaml
    - '{ genform || true ; } | grep -qF some-known-substring-of-the-help-text'
```
or
```yaml
    - '{ genform 2>&1 || true ; } | grep -qF some-known-substring-of-the-help-text'
```
in case the text is printed to the error stream.

Reply:
good idea. unfortunately the version number is not in the output, so I think that the current test is sufficient

Reply:
Yes, you don't have to do the `grep` thing since you have the other test. I just wanted to address your `meta.yaml` comment ;).